gateway/1.4/modules/ROOT/pages/flex-gateway-securing-local.adoc

```diff
@@ -0,0 +1,21 @@
+= Securing and Governing APIs in Local Mode
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+For Flex Gateway running in Local Mode, you apply API policies using xref:flex-local-configuration-reference-guide.adoc[local declarative configuration files]. You specify the location of these configuration files when first running Flex Gateway.
```

Comment:
```suggestion
For Flex Gateway running in Local Mode, you apply API policies using xref:flex-local-configuration-reference-guide.adoc[local declarative configuration files]. You specify the location of these configuration files when you install Flex Gateway.
```

Reply:
(I don't know if that's correct.)

Reply:
The config file location is specified when running the gateway, not installing.

Reply:
I changed it to:
`when you first run Flex Gateway.`